rudder-webapp/SOURCES/rudder-upgrade

```diff
@@ -379,6 +380,21 @@ upgrade_masterfiles() {
 # Check and update config
 ################################################################################
 
+upgrade_conf_file() {
+
+  STEP="Upgrade /opt/rudder/etc/rudder-web.properties"
+  CONF_FILE="/opt/rudder/etc/rudder-web.properties"
+
+  # - 7.0.0~alpha1 : Rename property rudder.community.port into rudder.policy.distribution.port.cfengine
+  OLD_PROPERTY_NAME="rudder.community.port"
+  NEW_PROPERTY_NAME="rudder.policy.distribution.port.cfengine"
+  ATTRIBUTESET=$(grep -c "^${NEW_PROPERTY_NAME}[ \t]*=" ${CONF_FILE} || true)
+  if [ ${ATTRIBUTESET} -eq 0 ]; then
```

Comment:
I think his should be an "if old_property is present" instead

Reply:
No, if the both are here (for ex, someone has added the new one in place or replacing the old one), then we don't want to do the sed, else we could have a duplicated property (and I'm not sure what happens in that case. First value taken? Last one?)